README.md

```diff
@@ -29,36 +29,10 @@ sudo swapon /dev/zram0
 
 ## Setup Rust for cross compilation
 
-I haven't figured this stuff out yet, I could use some help on these details.
+1. Make sure you have docker installed and running
+2. Get [cross](https://github.com/rust-embedded/cross) by running `cargo install cross`
+3. Build the Dockerfile in this folder with `docker build -t pinephone .`
```

Comment:
Slight error there. The image name in `Cross.toml` is `pinebuild`. Maybe change it to `pinephone` in `Cross.toml` as well.

Reply:
Thank you - it was late when I wrote that. Canged both to pinephone-build as that seems like a more descriptive name.